package.json

```diff
@@ -4,11 +4,13 @@
   "license": "MIT",
   "scripts": {
     "serve": "ts-node packages/infrastructure/src/lib/helpers/create-registry-files.ts && npx nx run-many -t serve",
-    "prepare-contribution": "npx nx g @cody-engine/contribution:prepare-contribution"
+    "prepare-contribution": "npx nx g @cody-engine/contribution:prepare-contribution",
+    "test": "npm run cucumber --prefix packages/be/"
```

Comment:
I've added a general test command to package.json in my latest commit. This will conflict with your test command here.  Instead of defining the backend test command in the root package.json, you can try to set it up directly in the be package and make it available for Nx to run.

This can be done by changing the "test" target in [packages/be/project.json](https://github.com/proophboard/cody-engine/blob/main/packages/be/project.json#L65)
or instead of changing the existing one for Jest, you could add another one for cucumber and call it "test-scenarios", "bdd" or similar.

The [serve target](https://github.com/proophboard/cody-engine/blob/main/packages/be/project.json#L40) should give you an idea how you can configure the target with a custom command (by using `nx:run-commands` as executor)